feat: add mentorship directory (#571)
asyncapi/community: mentorship/seasonofdocs/2022/README.md

# Status: Completed

# Season of Docs

> Google [Season of Docs](https://developers.google.com/season-of-docs) gives technical writers an opportunity to work with open source projects.

Reviewer: ```suggestion
Google [Season of Docs](https://developers.google.com/season-of-docs) allows technical writers to work with open-source projects.
```

## Program Maintainers

> - Alejandra Quetzalli ([@alequetzalli](https://github.com/alequetzalli)\): <https://twitter.com/QuetzalliAle>

Reviewer: ```suggestion
- Alejandra Quetzalli ([@alequetzalli](https://github.com/alequetzalli)\): <https://twitter.com/QuetzalliWrites>
```

- Lukasz Gornicki ([@derberg](https://github.com/derberg)\): <https://twitter.com/derberq>
## 2022 participation

> AsyncAPI has been [accepted as a mentoring organization](https://developers.google.com/season-of-docs/docs/2022/participants) in Season of Docs (GSoD) 2022!

Reviewer: ```suggestion
AsyncAPI was a [successful participant in Season of Docs (GSoD) 2022](https://developers.google.com/season-of-docs/docs/2022/participants)!
```

### ⌛ Timeline

> The project itself will take approximately 4-6 months to complete, depending on the different levels of knowledge from diverse technical writers (TW) that might get involved. (At AsyncAPI, we want to work with any TW, regardless of their years of experience. We have a passion for mentorship, and we do not wish to have a bar that would prevent any TW from contributing to our OSS Initiative. In fact, we look forward to potentially mentoring TW(s) who are completely new to tech and making them feel welcome!)

Reviewer: ```suggestion
The GSoD program runs for almost 6 months.
```

> For our 2 projects, we would like to request a minimum of 2 TWs, so that we can work on both the CLI/Tools and Generator Docs. 

Reviewer: ```suggestion
In 2022, GSoD at AsyncAPI had 2 projects and we hired 6 Technical Writers (TW).
```

> The timeline would look as following:

Reviewer: ```suggestion
The timeline ran as follows:
```

> - **May:** Orientation on how to contribute to AsyncAPI Inititiave, how Docs issues are organized, detail how we're migrating our CLI and Tools Docs, and assign good `first-time-tickets` to get each new TW contributor started. 

Reviewer: ```suggestion
- **May:** TWs received orientation on contributing to AsyncAPI Initiative, how Docs issues are organized, and assigned good `first-time-tickets` to get each new TW contributor started. 
```

> - **June - August:**	Each TW goes through designated issues marked for both first time contributors and work set aside for `GSoD 2022`. Each TW starts creating documentation for their individual issues assigned/selected. 

Reviewer: ```suggestion
- **June - August:**	Each TW went through designated issues marked for both first-time contributors and work set aside for `GSoD 2022`. Each TW started creating documentation for their issues assigned/selected. 
```

> - **September - October:** We determine if we're going to be able to complete both CLI and Tools Docs plus the Generator Docs, depending on how many TWs are in our group and how much they've been able to complete so far. We re-align priorities as needed and asses what is missing to reach our 2022 IA change goals for AsyncAPI Docs.

Reviewer: ```suggestion
- **September - October:** We determined if we could complete both projects. We re-aligned priorities as needed and assessed what is missing to reach our 2022 IA update goals for AsyncAPI Docs.
```

> - **November:**	Project completion and all contributors receive some swag! 

Reviewer: ```suggestion
- **November-mid December:**	Project completed, and all contributors received some swag! 
```

### 💸 Project budget

> We have set aside 2 mentors for now, for our 2 projects: improving our IA and re-structuring our Generator Docs. Should we be selected, AsyncAPI would like to request from Google a US $5000 budget for each project. For both projects, the request then totals for a $10,000 budget.  

Reviewer: ```suggestion
We had 2 mentors for our 2 projects: improving our IA and re-structuring our Generator Docs. AsyncAPI requested from Google a US $5000 budget for each project. For both projects, the request then totaled a $10,000 budget.  
```
